app/src/main/AndroidManifest.xml

```diff
@@ -2,60 +2,58 @@
     package="fr.free.nrw.commons">
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
-    <uses-permission android:name="android.permission.READ_SYNC_SETTINGS"/>
-    <uses-permission android:name="android.permission.READ_SYNC_STATS"/>
-    <uses-permission android:name="android.permission.WRITE_SYNC_SETTINGS"/>
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
I agree with this. :) Great work @psh , but next time it would be awesome if we could get the PRs split up a bit (one for style/formatting changes, etc) for easier reviewing.

I am still happy for this to be merged once it passes tests, though.